ci/azure-release.yml

```diff
@@ -1,13 +1,10 @@
 jobs:
   - job: ${{ parameters.name }}
     displayName: ${{ parameters.displayName }}
-    condition: and(succeeded(), contains(variables['Build.SourceBranch'], '${{parameters.branchref}}'))
+    # condition: and(succeeded(), contains(variables['Build.SourceBranch'], '${{parameters.branchref}}'))
```

Comment:
condition not needed?

Reply:
Condition is needed, commented out for testing.